src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -382,10 +382,7 @@ export const NavBar: React.FC = track(
                   Museums
                 </NavBarItemLink>
 
-                <NavBarItemLink
-                  href="https://nft.artsy.net"
-                  onClick={handleClick}
-                >
+                <NavBarItemLink href="/nft" onClick={handleClick}>
```

Comment:
Small thing-- I don't love using a vanity URL here, since it'll be confusing if we ever add an `/nft` path internal to Artsy.

Could we instead make this an ENV var?